src/pudl_archiver/archivers/classes.py

```diff
@@ -321,6 +323,7 @@ async def download_all_resources(
         for resource_chunk in resource_chunks:
             for resource_coroutine in asyncio.as_completed(resource_chunk):
                 resource_info = await resource_coroutine
+                self.logger.warning(f"{resource_info=}")
```

Comment:
Why `warning`? Is this leftover from testing?

Reply:
Yes indeed! Thank you.